### ISSUE TITLE
Bump protobuf and sqlite3 to resolve CVE issues

### ISSUE THREAD

**Issue by causten** [ROCmSoftwarePlatform/AMDMIGraphX#2341]
Component | Version | Latest version
-- | -- | --
protobuf | 3.11.0 | 24.3
sqlite3 | 3.17.0 | 3.43.2

**Comment by causten** [comment]
The sqlite3 seems pretty straightforward.   
For protobuf the requirements.txt lists it as 3.11 but then other requirements list it as 3.20.2  Since there are security vulnerabilities in 3.11 moving the minimum to 3.20.2 makes sense.  As for moving to 24.3 I worry about compatibilities with numpy. We may need to bump more then just protobuf to get things all compatible again

**Comment by ahsan-ca** [comment]
Component | Old Version | Updated version
-- | -- | --
protobuf | 3.11.0 | 3.19.0
sqlite3 | 3.17.0 | 3.43.2

The original version of Protobuf 3.11.0 had the security vulnerability CVE-2021-22570. Protobuf version 3.15.0 or greater fixes this vulnerability ([Source](https://www.cve.org/CVERecord?id=CVE-2021-22570)).
The recommendation was to update Protobuf version to 3.20.2 but that had a linker error.

I have updated Protobuf version to 3.19.0 as this version should not have the security vulnerability  CVE-2021-22570, and also does not have the linker error.
sqlite3 version has been updated to 3.43.2.
